### DISCUSSION TITLE
Customize bootstrap Kustomization location

### DISCUSSION THREAD

**Discussion by kiemlicz** [fluxcd/flux2#5018] [Q&A] [answered]
Hello,   
Is it possible to customize the location of files generated with `flux bootstrap github...` command?  
Currently they land in `$pathArgument/<namespace that can be customized>/*.yaml`

I want source and toolkit `kustomize.toolkit.fluxcd.io/v1` Kustomization to be present in different directory and all other manifests in different one.

**Comment by stefanprodan** [comment, marked as answer]
No this is not possible with `flux bootstrap`, but you can use the [Flux Operator](https://github.com/controlplaneio-fluxcd/flux-operator) to install and configure the Git sync, with the operator the Flux manifests are no longer stored in Git and you have a single YAML file with the Flux desired state.

**Reply by kiemlicz** [reply]
Thank you, 
additional question: is it safe to **add new** files into that generated directory modifying only the kustomization.yaml file?

**Reply by stefanprodan** [reply]
Yes, this is how bootstrap customization works, see the docs: https://fluxcd.io/flux/installation/configuration/boostrap-customization/
